src/containers/LoginFormContainer.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import SingleInput from '../components/SingleInput';
+import Container from '../components/Container';
+import { loginAuth, usernameCheck } from '../service/auth';
+//import {Link} from 'react-router';
+
+class LoginFormContainer extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {username: '', password: ''};
+
+        this.handleEvent = this.handleEvent.bind(this);
+        this.handleChange = this.handleChange.bind(this);
+        this.clearForm = this.clearForm.bind(this);
+        this.handleUsernameCheck = this.handleUsernameCheck.bind(this);
+    }
+
+    clearForm() {
+        this.setState({username : '', password: ''});
+    }
+
+    handleEvent(event) {
+        const init = {
+            method: 'post',
+            headers: {
+                "Content-type": 'application/json',
+            },
+            body: JSON.stringify({
+                username: this.state.username,
+                password: this.state.password,
+            }),
+        };
+
+        loginAuth(init);
```

Comment:
Suggest having loginAuth just take username/password and having all the headers/method details abstracted away in loginAuth.